docs/contributing.rst

```diff
@@ -324,6 +324,46 @@ Steps:
 7. Submit the PR.
 8. Did your tests pass on Travis? If they didn't, fix any errors.
 
+
+Updating certbot-auto and letsencrypt-auto
+==========================================
+Updating the scripts
+--------------------
+Developers should *not* modify the ``certbot-auto`` and ``letsencrypt-auto`` files
+in the root directory of the repository.  Rather, modify the
+``letsencrypt-auto.template`` and associated platform-specific shell scripts in
+the ``letsencrypt-auto-source`` and
+``letsencrypt-auto-source/pieces/bootstrappers`` directory, respectively.
+
+Building letsencrypt-auto-source/letsencrypt-auto
+-------------------------------------------------
+Once changes to any of the aforementioned files have been made, the
+``letesncrypt-auto-source/letsencrypt-auto`` script should be updated.  In lieu of
+manualy updating this script, run the build script, which lives at
+``letsencrypt-auto-source/build.py``:
+
+.. code-block:: shell
+
+   python letsencrypt-auto-source/build.py
+
+Running ``build.py`` will update the ``letsencrypt-auto-source/letsencrypt-auto``
+script.  Note that the ``certbot-auto`` and ``letsencrypt-auto`` scripts in the root
+directory of the repository will remain **unchanged** after this script is run.
```

Comment:
Thanks for the feedback @pde!

I've tried rewording this to take the active rather than the passive voice:
> Your changes will be propagated to these files during the next release of Certbot.

Let me know if you have any other suggested edits!